src/PostalCodes/PostalCode.cs

```diff
@@ -324,6 +395,26 @@ public virtual string ToHumanReadableString()
             return ToHumanReadableString(outputFormat);
         }
 
+        private string ExpandLowest()
+        {
+            if (_currentFormatType == FormatType.Short && _currentFormat.ShortExpansionAsLowestInRange != null)
+            {
+                return _backingPostalCode + _currentFormat.ShortExpansionAsLowestInRange;
+            }
+
+            return _backingPostalCode;
+        }
+
+        private string ExpandHighest()
+        {
+            if (_currentFormatType == FormatType.Short && _currentFormat.ShortExpansionAsHighestInRange != null)
+            {
+                return _backingPostalCode + _currentFormat.ShortExpansionAsHighestInRange;
+            }
+
+            return _backingPostalCode;
+        }
+
         /// <summary>
         /// Expands the postal code as lowest in range.
```

Comment:
Note: I didn't remove `ExpandPostalCodeAsLowestInRange` and `ExpandPostalCodeAsHighestInRange` because they're already exposed as public and appear to be used elsewhere (eg. when creating/updating the start and end of a `PostalCodeRange`).